src/pages/ReimbursementAccount/BankInfo/substeps/Manual.js

```diff
@@ -0,0 +1,121 @@
+import React, {useCallback} from 'react';
+import {Image, Text} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import Form from '@components/Form';
+import TextInput from '@components/TextInput';
+import useLocalize from '@hooks/useLocalize';
+import shouldDelayFocus from '@libs/shouldDelayFocus';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import exampleCheckImage from '@pages/ReimbursementAccount/exampleCheckImage';
+import {reimbursementAccountPropTypes} from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
+import * as ReimbursementAccountProps from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
+import subStepPropTypes from '@pages/ReimbursementAccount/subStepPropTypes';
+import getDefaultValueForReimbursementAccountField from '@pages/ReimbursementAccount/utils/getDefaultValueForReimbursementAccountField';
+import styles from '@styles/styles';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+const propTypes = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: reimbursementAccountPropTypes,
+
+    ...subStepPropTypes,
+};
+
+const defaultProps = {
+    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
+};
+
+const bankInfoStepKeys = CONST.BANK_ACCOUNT.BANK_INFO_STEP.INPUT_KEY;
+
+function Manual({reimbursementAccount, onNext}) {
+    const {translate, preferredLocale} = useLocalize();
+
+    const defaultValues = {
```

Comment:
does it make sense to wrap it in a `useMemo`?

Reply:
Honestly I dont think so. There are no expensive operations `perfomedgetDefaultValueForReimbursementAccountField` has only `lodashGet` that gets value for single field